hack/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM docker
```

Comment:
Add a comment describing what this is for. Otherwise it's not clear what a file called `hacker/Dockerfile` is for.